File: src/types/export.ts
// Tipi TypeScript per esportazione agenda

export interface ExportAgendaInput {
  data_inizio: string; // ISO string
  data_fine: string;   // ISO string
  operatrici_ids: string[];
  file_path: string;
}

export interface ExportResult {
  file_path: string;
  file_size: number;
}

File: src-tauri/src/models/export.rs
// Modelli per l'esportazione dell'agenda

use serde::{Deserialize, Serialize};

#[derive(Debug, Deserialize)]
pub struct ExportAgendaInput {
    pub data_inizio: String,
    pub data_fine: String,
    pub operatrici_ids: Vec<String>,
    pub file_path: String,
}

#[derive(Debug, Serialize)]
pub struct ExportResult {
    pub file_path: String,
    pub file_size: u64,
}
